### DISCUSSION TITLE
Timeout warning

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#726] [EXPERIMENTAL: Components, pages and patterns]
### Contribution purpose

Where a service has a time out (e.g. for security reasons), the user needs an indication of the time out.

### Examples

The following are examples of timeout warnings across MoJ products.

![Screenshot 2024-02-01 at 1 29 22 pm](https://github.com/user-attachments/assets/a3c856d6-67dd-48aa-b952-809d2c076cfc)
![Screenshot 2024-02-01 at 12 19 52](https://github.com/user-attachments/assets/b24c7a7a-6cb9-4908-a046-a588b186a0c8)
![Screenshot 2024-02-05 at 14 02 27](https://github.com/user-attachments/assets/91da61b6-2fa4-454d-bf46-21a2b36b71a1)
![Screenshot 2024-02-05 at 14 30 52](https://github.com/user-attachments/assets/4bed10a0-fc01-4ca9-87ff-3c63d6f46bb7)
![Screenshot 2024-02-05 at 14 31 03](https://github.com/user-attachments/assets/4924d2a2-f3e0-46a1-a3ed-702c24222a8d)

A pattern does exist in the Home Office's design system: https://design.homeoffice.gov.uk/patterns/stop-a-service-timing-out

**Comment by chrispymm** [comment]
There's been lots of chat on slack recently on this.  I probably have a component to contribute here 🤦 as I built one for MoJ Forms. It was pretty good, and passed our accessibility audit,
and with a few tweaks could be included.
